app/src/main/java/com/jerboa/ui/components/settings/account/AccountSettings.kt

```diff
@@ -125,7 +124,7 @@ fun SettingsForm(
     val sendNotificationsToEmail =
         rememberBooleanSettingState(luv?.local_user?.send_notifications_to_email ?: false)
     val sortTypeNames = remember {
-        MAP_SORT_TYPE_SHORT_FORM.values.map { ctx.getString(it) }
+        SortType.getSupportedSortTypes(siteViewModel.siteVersion()).map { ctx.getString(it.shortForm) }
```

Comment:
I'm a bit confused by which indexes won't lineup. The one index is the one in the supported sort types list, which is the other one?

What I was thinking about this is to add a second parameter to getSupportedSortTypes with the enableDownvotes boolean, but if I do this now I might as well implement the actual hiding too.

Reply:
That list of names gets fed to the dropdownsetting which sets the intsettingstate when you select a name. which is the index of that list

 Then in form it uses that index to select a sorttype. If we use the index to get it from the filtered list instead of all the sorttypes then we can pass sortType.ordinal to the form. This way it will always have the right index.

Reply:
Maybe it's a bit of a clunky solution, I made it so that the sort type returned by the backend is matched to the supported types using indexOf, and then I pass the supported sort type to the form using the index returned by the dropdown.

What do you think?